xilem/src/driver.rs

```diff
@@ -88,18 +88,15 @@ impl RawProxy for MasonryProxy {
         }
     }
     fn dyn_debug(&self) -> &dyn std::fmt::Debug {
-        self
+        #[derive(Debug)]
+        struct MasonryProxy;
+        &MasonryProxy
     }
```

Comment:
Wait, what?

Reply:
The `dyn Fn` doesn't implement `Debug` ... but yeah I can see how this is confusing. Changed to a manual `Debug` impl with `finish_non_exhaustive`.

Reply:
No, I mean, why is a struct defined in the function body?

Reply:
Well it was never used outside this function. No worries. An explicit Debug impl is better anyway because it has the `..` indicating that it's not exhaustive.